src/questions/fjord-foundry.yaml

```diff
@@ -0,0 +1,170 @@
+      key: D
+- uuid: adee7b00-ae1d-431c-82fb-4362cc778bce
+  type: SingleChoice
+  content: What feature of Fjord Foundry makes it resistant to sniper bots?
+  hint: NoHint
+  explanation: Fjord Foundry has implemented a built-in high-low price discovery mechanism, which makes it resistant to sniper bots. This allows users to join in whenever they feel the price is fair.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: A virtual queueing system for distributing tickets fairly
+      key: A
+    - content: An automated system for identifying suspicious behaviour
+      key: B
+    - content: A blockchain-based authentication system for tickets
+      key: C
+    - content: A built-in high-low price discovery mechanism
+      key: D
+- uuid: 9d94333d-1466-4fd2-a5c4-9e661f781fa1
+  type: SingleChoice
+  content: Is there a minimum or maximum allocation for LBP participants?
+  hint: NoHint
+  explanation: Unlike other platforms, LBPs do not require whitelists, hard caps, or listing requirements. There is no minimum or maximum allocation and LBP participants are free to choose how much they want to buy.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: Yes, there is a minimum and maximum allocation.
+      key: A
+    - content: No, there is no minimum or maximum allocation.
+      key: B
+    - content: There is a minimum allocation, but no maximum allocation.
+      key: C
+    - content: There is a maximum allocation, but no minimum allocation.
+      key: D
+- uuid: 39b074b3-54f5-4fce-933e-f70a4441d933
+  type: SingleChoice
+  content: What is a key feature of Liquidity Bootstrapping Pools (LBPs)?
+  hint: NoHint
+  explanation: Some of the key features of LBPs are Permissionless participant, Price discovery, Capital efficiency and Fair distribution.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: Permissionless participation
+      key: A
+    - content: Price discovery
+      key: B
+    - content: Capital efficiency
+      key: C
+    - content: All of the above
+      key: D
+- uuid: b59363b6-e306-4c35-b611-eef7316d830a
+  type: SingleChoice
+  content: What happens during the Redemption Pool period?
+  hint: NoHint
+  explanation: Redemption pools are special pools that allow projects to progressively distribute a token over a defined time period in exchange for a specified base token (In some cases this could be releasing a governance token in exchange for LP). During the Redemption Pool period, users can exchange the base token for the project's token, which contributes to the collective decision of the asset's fair value.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: Users pay for a governance token in exchange for liquidation tokens.
+      key: A
+    - content: Users can exchange the project's token for the base token.
+      key: B
+    - content: Users can exchange the base token for the project's token.
```

Comment:
Include "progressively distribute a token over a defined time period" as that is very important point